### DISCUSSION TITLE
ADC parameters after reset

### DISCUSSION THREAD

**Discussion by Schwenkner** [arendst/Tasmota#10856] [Q&A] [answered]
May be I'm only missed something.

I've found in the docs I can provide params to te ADC.
I'm using this one "AdcParam 6, 0, 1023, 0, 3200" and it works. The display changes from "Analog0" to "Range" and the correct value is also sent via MQTT.

But after reset or deep sleep I get the old analog values, not the range.

I tried already "savedata 1", but didn't help. Am I doing something wrong or isn't that feature available (yet)?

**Comment by sfromis** [comment, marked as answer]
You also need to configure the analog gpio17 for range, before the range function of AdcParam works properly.

**Reply by Schwenkner** [reply]
Thx, that's what I've missed. Now it works.

**Comment by barbudor** [comment]
Hi @Schwenkner 
I'm using it on a battery powered sensor with Deepsleep, so I can insure that it works.

What binary are you using ? A standard binary from Github or ota.tasmota.com ?
Or a custom binary ? If so, how are you compiling it ? (Gitpod, TasmoCompiler, VSC, ...) ?
